aws_connect.py

```diff
@@ -139,7 +164,7 @@ async def login_for_access_token(
     user = authenticate_user(form_data.username, form_data.password)
     if not user:
         raise HTTPException(
-            status_code=state.HTTP_401_UNAUTHORIZED,
+            status_code=401,
```

Comment:
that should be `status.HTTP_401_UNAUTHORIZED` as in `from fastapi import status`